gazebo/gazebo7/gzweb7/Dockerfile

```diff
@@ -17,13 +17,12 @@ RUN apt-get update && apt-get install -q -y \
     nodejs-legacy \
     npm \
     pkg-config \
-    psmisc \
-    xvfb\
```

Comment:
Thanks, looks like gzweb's yaml wasn't updated to include the add packages we tried to solve:
https://github.com/osrf/docker_images/issues/43